### PR TITLE
CU-86c1djzn9 - fix(ci): Update tests after removing installationId authentication type from BE endpoints 

### PR DESCRIPTION
**Pull request by Moran-k**
Fix CI to use relevant test API keys

### CODE REVIEW

File: .buildkite/tests/values_capabilities_test.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def wait_for_metrics(container_name, pod_name):
                f"clusterName={CLUSTER_NAME}&namespace={NAMESPACE}&podName={pod_name}&"
                f"fromEpoch={start_time}&pageSize=1&timeWindow=1d&endTime={end_time}&"
                f"aggregationTypes=p96&aggregationTypes=p99")
-        response = query_backend(url)
+        response = query_backend(url, agent_api_key=False)
```
Comment (nirbenator) [inline .buildkite/tests/values_capabilities_test.py:23]: i see that we also `query_backend` in `test_network_mapper`
why dont we change to agent_api_key_false there?

**Review by nirbenator — APPROVED** [review]
lgtm - just a small question

File: .buildkite/tests/values_capabilities_test.py (hunk 1 above)
Comment (Moran-k) [reply .buildkite/tests/values_capabilities_test.py:23]: I changed the default to false, so basically, we are not using agent-api anymore

**Pull request merged**